cli/azd/cmd/auth_login.go

```diff
@@ -118,6 +119,14 @@ func (lf *loginFlags) Bind(local *pflag.FlagSet, global *internal.GlobalCommandO
 	)
 	// ensure the flag behaves as a common boolean flag which is set to true when used without any other arg
 	f.NoOptDefVal = "true"
+	b := local.VarPF(
+		&lf.manualBrowsing,
+		"no-browser",
```

Comment:
Overall feedback (that I'd love to hear from @ellismg as well):

- If we want a "quick to keyboard" solution, adding another flag "--no-browser" is indeed most discoverable. The tradeoff here is feature complexity (both for users and maintainers).
- Other alternatives that was quick to my mind:
   - Always print out the action of opening a browser -- after all, isn't opening a browser a somewhat worthwhile action to be logged?
   - Make a slight code tweak to `$BROWSER` cmd invocation to attach stdout/stderr. Document in help with a footnote:
       - In environments where the browser isn't available, `$BROWSER` can be overridden to print the login link: : `BROWSER='echo' azd auth login`.
       - I tested this and it worked as expected.